File: codeforces/131A/go/131a.go
package main

import "fmt"

func is_upper(ch byte) bool {
    return ch >= 'A' && ch <= 'Z'
}

func main() {
    var s string
    fmt.Scan(&s)

    var count int = 0
    for i := 1; i < len(s); i++ {
        if is_upper(s[i]) {
            count++
        }
    }

    var result string = ""
    if count == len(s) - 1 {
        for i := 0; i < len(s); i++ {
            if is_upper(s[i]) {
                result += string(s[i] - 'A' + 'a')
            } else {
                result += string(s[i] - 'a' + 'A')
            }
        }
    } else {
        result = s
    }

    fmt.Println(result)
}

File: codeforces/131A/py/131a.py
import sys

def is_upper(ch):
    return ch >= 'A' and ch <= 'Z'

s = input()
count = 0
for i in range(1, len(s)):
    if is_upper(s[i]):
        count = count + 1

result = ""
if (count == len(s) - 1):
    for i in range(len(s)):
        if (is_upper(s[i])):
            result = result + (s[i].lower())
        else:
            result = result + (s[i].upper())
else:
    result = s

print(result)
